### PR TITLE
Add js steps to US Sen yamls; edit topic options for Gov yaml

### PR DESCRIPTION
**Pull request by OdeclasV**
- Adding js steps to US Sen yamls to find and replace phrase that I believe is messing with msg deliverability. 
- Editing topic options for Governor yaml to account for toppic mapping

### CODE REVIEW

File: members/B001261.yaml [hunk 1]
```diff
@@ -41,13 +40,11 @@ contact_form:
...
           required: true
-          options:
-            blacklist: ":’"
     - javascript:
         - name: Message
-          selectors: [ "#field_5722C424-8601-4ABF-A8D1-AC11B9C45F87" ]
+          selectors: [ "#field_40C203DD-A384-408D-84AB-26D1471756D8" ]
           values: [ "$MESSAGE" ]
-          commands: ["elements[0].value = values[0].replace(/\bset\b/ig,'determine')"]
+          commands: ["elements[0].value = values[0].replace(/\\b(set forth in the bill)\\b/ig,'')"]
```
Comment (vramoscoreas) [inline members/B001261.yaml]: So we're just replacing the 'set' text with an empty string here, is that correct?

Comment (OdeclasV) [reply members/B001261.yaml]: yeah, we're replacing `set forth in the bill` , to be exact

Comment (vramoscoreas) [reply members/B001261.yaml]: Cool 👍, just wanted to make sure I was reading that right. It looked like previously we we're just replacing the word 'set' with 'detemine' so also wanted to make sure the change was intentional.

Comment (OdeclasV) [reply members/B001261.yaml]: yup yup. Thank you!

**Pull request merged**